core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -32,29 +32,84 @@ import com.iteneum.designsystem.theme.Drab
 import com.iteneum.designsystem.theme.LPTypography
 
 /**
- * Created [LpPostCard]
+ * Created [LpGenericCard]
  *
  * @param modifier to modify an specific property of the card
- * @param content as a high order function
+ * @param title as an example 'Current balance'
+ * @param details this is the clickable option
+ * @param accountNumber the showed String number
+ * @param currency if the number represents currency for example current balance
+ * @param onTextClick as a high order function
  *
- * @author Daniel Roldan
+ * @author Juan R Islas
  */
 @Composable
-fun LpBasicCard(
+fun LpGenericCard(
     modifier: Modifier = Modifier,
-    content: @Composable (ColumnScope.() -> Unit)
+    title: String,
+    details: String,
+    accountNumber: String,
+    currency: Boolean = false,
+    onTextClick: () -> Unit
 ) {
     Card(
         modifier = modifier
             .fillMaxWidth()
```

Comment:
I recommend you remove this property `.fillMaxWidth()` and send it in the parameter modifier

Reply:
Hi, comment resolved